Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[workspace]
+members = [
+  "objects",
+]
+
+[profile.release]
+codegen-units = 1
```

Comment:
I'd rather keep it here. As far as I can tell, it doesn't impact much in terms of compiler performance. But it may prevent some confusions in benchmarking in the future when some people enable this and others don't.